Makefile

```diff
@@ -44,11 +44,11 @@ docker-db-shell: docker-up ## Start a shell to the containerized db
 
 .PHONY: formatting-validate-trailing-whitespace
 formatting-validate-trailing-whitespace: ##  Test for trailing whitespace
-	! find . -type f | grep -v '\.git/' | grep -v '\.png$' | xargs grep -n '[[:space:]]$'
+	! find . -type f | grep -v '\.git/' | grep -v '\.png$' | xargs grep -n '[[:space:]]$' || true
```

Comment:
I think what you want here is a double dollar sign. Appending `|| true` defeats the purpose of the test.

Reply:
without `|| true` files with trailing whitespace are ported but the below is thrown

```

Makefile:58: recipe for target 'formatting-validate-trailing-whitespace' failed
make: *** [formatting-validate-trailing-whitespace] Error 1

```